raiko-guests/sgx/script/src/lib.rs

```diff
@@ -0,0 +1,253 @@
+#![cfg(feature = "enable")]
+use std::{
+    env,
+    fs::{copy, create_dir_all, remove_file, File},
+    path::PathBuf,
+    process::Output,
+    str,
+};
+
+use once_cell::sync::Lazy;
+use raiko_lib::input::{GuestInput, GuestOutput};
+use serde::{Deserialize, Serialize};
+use serde_json::Value;
+use serde_with::serde_as;
+use tokio::{process::Command, sync::OnceCell};
+
+#[serde_as]
+#[derive(Clone, Debug, Serialize, Deserialize)]
+pub struct SgxParam {
+    pub instance_id: u64,
+    pub input_path: Option<PathBuf>,
+}
+
+#[derive(Default, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct SgxResponse {
+    /// proof format: 4b(id)+20b(pubkey)+65b(signature)
+    pub proof: String,
+    pub quote: String,
+}
+
+pub const ELF_NAME: &str = "sgx-bin";
+pub const INPUT_FILE_NAME: &str = "input.bin";
+pub const CONFIG: &str = "../../raiko-guests/sgx/config";
+
+static GRAMINE_MANIFEST_TEMPLATE: Lazy<OnceCell<PathBuf>> = Lazy::new(OnceCell::new);
+static INPUT_FILE: Lazy<OnceCell<PathBuf>> = Lazy::new(OnceCell::new);
+static PRIVATE_KEY: Lazy<OnceCell<PathBuf>> = Lazy::new(OnceCell::new);
+static ATTESTATION_TYPE: Lazy<OnceCell<PathBuf>> = Lazy::new(OnceCell::new);
+static QUOTE: Lazy<OnceCell<PathBuf>> = Lazy::new(OnceCell::new);
+static USER_REPORT_DATA: Lazy<OnceCell<PathBuf>> = Lazy::new(OnceCell::new);
```

Comment:
Why lazy? Seems to complicate the code quite a bit and these are very cheap operations so don't really see any reason to try and optimize this.

Reply:
Cuz it complains:
```
cannot call non-const fn `tokio::sync::OnceCell::<PathBuf>::new` in statics
calls in statics are limited to constant functions, tuple structs and tuple variants
consider wrapping this expression in `Lazy::new(|| ...)` from the `once_cell` crate: https://crates.io/crates/once_cellrustc[Click for full compiler diagnostic](rust-analyzer-diagnostics-view:/diagnostic%20message%20%5B0%5D?0#file%3A%2F%2F%2Fmnt%2Fhome%2Fubuntu%2Fraiko-brecht%2Fraiko%2Fraiko-guests%2Fsgx%2Fscript%2Fsrc%2Flib.rs)
```
I want to make them static variables cuz it's easy to see, but I don't wanna hardcode the address cuz inside closure it evaluates the working directory.